TACC/tup-ui
# fix: tup-615 inconsistent header spacing - left & vertical

## Overview

Make header spacing consistent.

## Related

- [TUP-615](https://jira.tacc.utexas.edu/browse/TUP-615)

## Changes

- #338
- #340
- #341
- #342
- [fix: project ticket grid too much middle space (`f027640`)](https://github.com/TACC/tup-ui/pull/339/commits/f027640b0a3991ba770f8758450b6e0d02408185)

## Testing

1. Open all sections.
2. Verify all header text has the same space before and above.
3. Verify, in Dashboard section, on large window (or with small text):
    - no excess space above "Dashboard" header text
    - no excess space between "Active Projects" and "My Tickets" table

## UI

| | | |
| - | - | - |
| <img width="400" alt="dashboard" src="https://github.com/TACC/tup-ui/assets/62723358/f65da7c6-f28a-4a67-b3df-f070b6dfa609"> | <img width="400" alt="proj alloc" src="https://github.com/TACC/tup-ui/assets/62723358/59c18dd8-9fe9-46fb-9fd6-66c7e43c3ce9"> | <img width="400" alt="project" src="https://github.com/TACC/tup-ui/assets/62723358/958e1658-e639-4ed7-b20c-2eff86b709ec">
<img width="400" alt="tickets" src="https://github.com/TACC/tup-ui/assets/62723358/1e87b38e-104d-4da0-9261-c17189c705a9"> | <img width="400" alt="system" src="https://github.com/TACC/tup-ui/assets/62723358/4322347e-aa34-4a7f-b2d7-5c3598c902c5"> | <img width="400" alt="account" src="https://github.com/TACC/tup-ui/assets/62723358/ecc29287-e607-43fc-b4ad-381a9f496701"> |

| dashboard layout |
| - |
| <img width="1200" alt="dashboard - full" src="https://github.com/TACC/tup-ui/assets/62723358/fb51093d-37cf-4b6b-8791-4b8713216987"> |

The cut-off System Status "Running Jobs" is caused by #312, not this.
